nativescript-charter.md

```diff
@@ -0,0 +1,106 @@
+# NativeScript Charter
+
+## Section 0. Guiding Principle
+
+NativeScript is part of the OpenJS Foundation. The project operates transparently, openly, collaboratively, and ethically. Project proposals, timelines, and status must not merely be open, but also easily visible to outsiders.
+
+## Section 1. Scope
+
+NativeScript brings native platform APIs directly into the JavaScript runtime, enriched with robust TypeScript typings, ensuring a seamless and powerful development experience. As an open-source framework for building applications across iOS, visionOS, Android, macOS, and more, it uniquely blends familiar web development paradigms—such as CSS styling and template-driven views—with popular native languages including Swift, Kotlin, Objective-C, and Java. This integration offers developers unparalleled flexibility and ease.
+Its intuitive CLI further simplifies interaction with native platform development processes, making native development approachable for JavaScript developers.
+
+### 1.1: In-scope
+
+Section Intentionally Left Blank
+
+### 1.2: Out-of-Scope
+
+Section Intentionally Left Blank
+
+## Section 2. Relationship with OpenJS Foundation CPC.
+
+Most large, complex open source communities have both a business and a technical governance model. Technical leadership for the projects within the OpenJS Foundation is delegated to the projects through their project charters by the OpenJS Cross Project Council (CPC). In the case of NativeScript, it is delegated to the NativeScript Technical Steering Committee ("TSC"). OpenJS Foundation's business leadership is the Board of Directors (the "Board").
+
+This charter can only be amended with the approval of the CPC.
+
+## Section 3. Establishment of the TSC
+
+TSC memberships are not time-limited. There is no maximum size of the TSC.
```

Comment:
```suggestion
TSC memberships are reviewed every 2 years. Upon review, the TSC meeting attendance of members as well as contributions to the project and community are considered. There is no maximum size of the TSC.
```

Good idea @ljharb, I think it makes sense in NativeScript's case to have membership reviews every 2 years as it is a large project covering wide surface are and it does receive diverse feedback and participation within broad windows.

Reply:
Solid point :) I think we may want to follow a similar min/max size as Node if there’s a good precedent on that within their team.